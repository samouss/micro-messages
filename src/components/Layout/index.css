.Layout {
  display: flex;
  flex-direction: row;
  height: 100vh;
  background-color: #FFFFFF;
}
