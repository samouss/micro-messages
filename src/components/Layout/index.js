// @flow

import React from 'react';
import './index.css';

type Props = {
  children: React.Element<*>,
};

const Layout = ({ children }: Props) => (
  <div styleName="Layout">
    {children}
  </div>
);

export default Layout;
