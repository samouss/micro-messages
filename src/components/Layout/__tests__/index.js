// @flow

import React from 'react';
import { shallow } from 'enzyme';
import Layout from '../index';

describe('<Layout />', () => {
  it('expect to render the component correctly', () => {
    const component = shallow(
      <Layout>
        <div>Some content</div>
      </Layout>,
    );

    expect(component).toMatchSnapshot();
  });
});
