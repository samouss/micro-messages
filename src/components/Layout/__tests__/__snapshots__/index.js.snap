// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> expect to render the component correctly 1`] = `
<div
  styleName="Layout"
>
  <div>
    Some content
  </div>
</div>
`;
